### PR TITLE
DP-1370 Radio buttons and checkboxes more visible in feedback forms

### PR DESCRIPTION
**Pull request by mrossi113**
Accessibility fix to make the radio buttons and check boxes more visible on feedback form. The border color for these were changed to #757575 per Sarena and Harlan request in the comments. These checkbox and radio buttons borders were changed from border-color: $c-bd-input; to border-color: #757575;

JIRA ticket: https://jira.state.ma.us/browse/DP-1370

To test:
Compare the radio button with the current pilot.mass.gov/topics/health-social-services feedback form.
Inspect the radio button and change the .ma__input-radio border color to #757575
To compare the check box go to http://localhost:3000/?p=pages-pilot-transition vs current site https://pilot.mass.gov/leaving-pilot?continueURL=http://www.mass.gov/dor/child-support/court-information/paternity-establishment/

### CODE REVIEW

**Review by gender404 — COMMENTED** [review]
We have been trying to not use colors directly in SASS files to help maintain consistency.  To make this #757575 you should either create a new variable $c-bd-input-dark or change the value of $c-bd-input to #757575.

Variables are named using a pattern:
c => color variable
bd => applies to borders
input => descriptive name
dark => optional modifier

I'm going to go out on a limb here and say that we are most likely going to need to change the border color on all inputs to #757575 for readability.  

You just need to search the scss file to determine what other items are using this variable before updating.

**Comment by reenybeeny** [comment]
@legostud / @mrossi113 - it was decided not to change the border color on inputs or dropdowns at this time, so the change should only apply to radio buttons and checkboxes.

**Comment by mrossi113** [comment]
@legostud / @reenybeeny  Okay I will create a new variable in the SASS file with #757575 for just these two radio buttons and checkboxes border color.

**Comment by isaacchansky** [comment]
Probably a good idea to darken the borders, but the real purpose of this ticket is to fix the issue of identifying _focus_.

So the user-story here is "as someone with potential visual impairment, it should be easy for me to tell where my keyboard focus is." 

It looks like the focus styles haven't changed, they're still:
```
outline-color: -webkit-focus-ring-color;
```

**Comment by gender404** [comment]
The radio button is missing the outline.  You can try adding that to make it more visible.

.ma__input-radio input:focus ~ label:before {
  outline: -webkit-focus-ring-color auto 5px;
}

.ma__input-radio label:before {
 ... 
 overflow: hidden;
  ...
}

File: styleguide/source/assets/scss/06-theme/00-base/_colors.scss [hunk 1]
```diff
@@ -34,6 +35,6 @@ $c-bg-section: $c-gray-f2f2f2;
 
 $c-bd-divider: $c-gray-dcdcdc;
 $c-bd-input: $c-gray-dcdcdc;
-
+$c-bd-input-dark: $c-gray-757575;
```
Comment (gender404) [inline styleguide/source/assets/scss/06-theme/00-base/_colors.scss:38]: nicely done

File: styleguide/source/assets/scss/01-atoms/_forms.scss [hunk 2]
```diff
@@ -75,6 +76,7 @@ label {
         left: 0;
       text-align: center;
       width: 2em;
+      overflow: hidden;
```
Comment (gender404) [inline styleguide/source/assets/scss/01-atoms/_forms.scss:79]: I don't believe this is needed here.  I added it to the radio button due to the font-size being larger than the height.  

If you want to see an interesting outline shape, use the inspector tools to turn overflow:hidden off on the radio button before element.

File: styleguide/source/assets/scss/06-theme/01-atoms/_forms.scss [hunk 3]
```diff
@@ -106,6 +106,7 @@ label {
      */
     @media (-webkit-min-device-pixel-ratio:0) {
       outline-color: -webkit-focus-ring-color;
+      outline: -webkit-focus-ring-color auto 5px;
```
Comment (gender404) [inline styleguide/source/assets/scss/06-theme/01-atoms/_forms.scss:109]: outline includes outline-color so you don't need both.

File: styleguide/source/assets/scss/06-theme/01-atoms/_forms.scss [hunk 4]
```diff
@@ -132,6 +133,7 @@ label {
...
```
Comment (gender404) [inline styleguide/source/assets/scss/06-theme/01-atoms/_forms.scss:136]: outline includes outline-color so you don't need both.

**Comment by mrossi113** [comment]
Talked with @WheresHJ about the border color and the focus for checkboxes and radio buttons. From the discussion was to keep the #757575 as the border color as well as the suggestion from @legostud 

`.ma__input-radio input:focus ~ label:before {
outline: -webkit-focus-ring-color auto 5px;
}
.ma__input-radio label:before {
...
overflow: hidden;
...
}`

@isaacchansky Harlan said he would talk with you about these changes.

**Review by gender404 — CHANGES_REQUESTED** [review]
Just a few changes

**Comment by isaacchansky** [comment]
Cool this looks good for now. We'll plan to circle back later and look at all focus states holistically to ensure that we're consistent and deliberate about how the whole system works together.

File: styleguide/source/assets/scss/01-atoms/_forms.scss (hunk 2 above)
Comment (mrossi113) [reply styleguide/source/assets/scss/01-atoms/_forms.scss:79]: Yes the `overflow:hidden` was not needed for the checkboxes only the radio buttons. Made the change.

File: styleguide/source/assets/scss/06-theme/01-atoms/_forms.scss (hunk 3 above)
Comment (mrossi113) [reply styleguide/source/assets/scss/06-theme/01-atoms/_forms.scss:109]: Removed the outline-color

File: styleguide/source/assets/scss/06-theme/01-atoms/_forms.scss (hunk 4 above)
Comment (mrossi113) [reply styleguide/source/assets/scss/06-theme/01-atoms/_forms.scss:136]: Removed the outline-color

**Comment by isaacchansky** [comment]
Sorry, closed this too early: adjustments @legostud suggested are being done here: https://github.com/massgov/mayflower/pull/365/files

**Pull request merged**